modules/oceanbase/src/main/java/org/testcontainers/oceanbase/OceanBaseCEContainer.java

```diff
@@ -64,19 +127,31 @@ public String getDatabaseName() {
 
     @Override
     public String getUsername() {
-        // In OceanBase, the jdbc username is related to the name of user, tenant and cluster,
-        // if a tenant name other than the default value 'test' is used, you should manually
-        // construct the jdbc username by yourself.
-        return DEFAULT_USERNAME + "@" + DEFAULT_TEST_TENANT_NAME;
+        return DEFAULT_USER + "@" + tenantName;
     }
 
     @Override
     public String getPassword() {
-        return DEFAULT_PASSWORD;
+        return password;
     }
 
     @Override
     protected String getTestQueryString() {
         return "SELECT 1";
     }
+
+    public OceanBaseCEContainer withMode(String mode) {
```

Comment:
Why not using the enum?

```suggestion
    public OceanBaseCEContainer withMode(Mode mode) {
```